templates/client-podsecuritypolicy.yaml

```diff
@@ -41,7 +41,6 @@ spec:
   hostIPC: false
   hostPID: false
   runAsUser:
-    # Require the container to run without root privileges.
```

Comment:
I'm curious, are you deleting these comments because of duplication?

Reply:
I'm deleting them because they're incorrect. `RunAsAny` means the container can run as anything. It doesn't prevent root. `MustRunAs` or `MayRunAs` can be used to prevent root.